### PR TITLE
Add pr link to repository card

### PR DESCRIPTION
**Pull request by pfeuffer**
## Proposed changes

Add a link to the pull requests inside the repository card overview

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] PR is well described
- [x] Target branch is not master (in most cases develop should bet the target of choice) 
- [x] Code does not conflict with target branch
- [ ] New code is covered with unit tests
- [x] CHANGELOG.md updated
- [ ] Documentation updated (only necessary for new features or changed behaviour)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: src/main/js/index.tsx [hunk 1]
```diff
@@ -80,5 +81,9 @@ binder.bind("repos.branch-details.information", ({ repository, branch }) => (
   <CreatePullRequestButton repository={repository} branch={branch} />
 ));
 
+binder.bind("repository.card.quickLink", ({ repository, repositoryLink }) => (
+  <RepositoryPullRequestCardLink repository={repository} repositoryLink={repositoryLink} />
+));
+
```
Comment (sdorra) [inline src/main/js/index.tsx]: I think this is the same as `binder.bind("repository.card.quickLink", RepositoryPullRequestCardLink)`.

Comment (sdorra) [reply src/main/js/index.tsx]: I would use a precondition to avoid rendering the extension, if no pr link is available.

Comment (pfeuffer) [reply src/main/js/index.tsx]: The guard is inside RepositoryPullRequestCardLink. Isn't this the same?

Comment (sdorra) [reply src/main/js/index.tsx]: No, it is not. With a guard inside the RepositoryPullRequestCardLink, the component it self is rendered. With a precondition the RepositoryPullRequestCardLink is not rendered at all.

**Pull request merged**